feat: add additional release links to the release (#282)
semantic-release/github: README.md

The `releasedLabels` ```['released<%= nextRelease.channel ? ` on @\${nextRelease.channel}` : "" %> from <%= branch.name %>']``` will generate the label:

> released on @next from branch next

#### addReleases

> When set this is option will append all additional releases except the github release to the body of the github release.

Reviewer: I'd rephrase the sentence, now that it can be added to the top or bottom

```suggestion
Add links to other releases to the GitHub release body.
```

> Valid values for this option are `false`, `top` or `bottom` where the releases will be added on the top when set to `top` and on the bottom when set to `bottom`.

Reviewer: I think `top` and `bottom` are self-explanatory enough? I would add quotes to make clear that `false` should not be a string.

```suggestion
Valid values for this option are `false`, `"top"` or `"bottom"`.
```
